packages/terra-site/src/examples/responsive-element/Index.jsx

```diff
@@ -9,20 +9,44 @@ import { version } from 'terra-responsive-element/package.json';
 import ResponsiveElementSrc from '!raw-loader!terra-responsive-element/src/ResponsiveElement';
 
 // Example Files
-import Responsive from './Responsive';
-import AnimatedResponsiveElement from './AnimatedResponsiveElement';
+import ResponsiveExample from './ResponsiveExample';
 
 const ResponsiveElementExamples = () => (
   <div>
     <div id="version">Version: {version}</div>
     <Markdown id="readme" src={ReadMe} />
     <PropsTable id="props" src={ResponsiveElementSrc} />
     <h2>Responsive to Parent</h2>
-    <AnimatedResponsiveElement />
-    <Responsive />
+    <h3>All Breakpoints</h3>
+    <ResponsiveExample
+      defaultElement={<div>Default</div>}
+      tiny={<div>Tiny</div>}
+      small={<div>Small</div>}
+      medium={<div>Medium</div>}
+      large={<div>Large</div>}
+      huge={<div>Huge</div>}
+    />
+    <br />
+    <h3>Default and Medium Breakpoints</h3>
+    <ResponsiveExample
+      defaultElement={<div>Default</div>}
+      medium={<div>Medium</div>}
+    />
+    <br />
+    <h3>Only Medium</h3>
+    <h5>Note: Does not render elements when smaller than medium.</h5>
```

Comment:
Sorry, these need to be h4. The only reason I care is when we use accessibility tools this will be an issue. We need to use heading tags for semantic structuring, not styling. h6 must be used after h5, h5 must be used after h4, etc.

http://webaim.org/techniques/semanticstructure/#correctly